go.mod

```diff
@@ -0,0 +1,38 @@
+module github.com/canonical/identity-platform-admin-ui
+
+go 1.20
```

Comment:
stick with `1.19` unless u wanna change here https://github.com/canonical/identity-platform-admin-ui/pull/1/files#diff-2e1e05b21277cb5d4b719e39e9005188dc9f4efae49c301c1af5e703df86d45aR13 and here https://github.com/canonical/identity-platform-admin-ui/pull/1/files#diff-67c464bbed217d3d422ddd2db32d89360c18ce8584542858e61f3f842c6fb432R27

Reply:
Good catch